actionpack/lib/action_controller/metal/implicit_render.rb

```diff
@@ -2,8 +2,14 @@ module ActionController
   module ImplicitRender
     def send_action(method, *args)
       ret = super
-      default_render unless performed?
-      ret
+      return ret if performed?
+
+      if template_exists?(method, _prefixes, variants: request.variant || [])
```

Comment:
@georgeclaghorn that's what I was thinking, :+1:.

@sb8244 can you link to the line which is erring?